app/bundles/AssetBundle/Assets/js/asset.js

```diff
@@ -28,7 +28,8 @@ Mautic.updateRemoteBrowser = function(provider, path) {
 
                 mQuery('.remote-file-search').quicksearch('#remoteFileBrowser .remote-file-list a');
             } else {
-                // TODO - Add error handler
+                const flashMessage = Mautic.addErrorFlashMessage(Mautic.translate('mautic.asset.update.remote.browser.error'));
+                Mautic.setFlashes(flashMessage);
```

Comment:
Can we use the error message from the flash message generated by the server instead of doing this in JS?